smart-send-logistics/block.json

```
@@ -0,0 +1,13 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 3,
+	"name": "create-block/shipping-workshop",
+	"version": "1.0.0",
+	"title": "Shipping Workshop",
```

The [title](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/#title) should be the displayed name.

Consider something like:

```suggestion
	"title": "Pick-up Point checkout",
```